test/smooth_test.cpp

```diff
@@ -268,18 +268,6 @@ TEST(Smooth, Csaszar) {
     ExportOptions options;
     options.faceted = false;
     options.mat.roughness = 0.1;
-
-    options.mat.vertColor.resize(csaszar.NumVert());
-    const vec4 blue(0, 0, 1, 1);
-    const vec4 yellow(1, 1, 0, 1);
-    for (size_t tri = 0; tri < csaszar.NumTri(); ++tri) {
-      for (int i : {0, 1, 2}) {
-        const vec3& uvw = {0.5, 0.5, 0.0};
-        const double alpha = std::min(uvw[0], std::min(uvw[1], uvw[2]));
-        options.mat.vertColor[out.triVerts[3 * tri + i]] =
```

Comment:
do we have color for this now?

Reply:
No, but it also wasn't working before - I realized this was leftover code that wasn't really doing much.